apps/src/music/views/MusicLabView.tsx

```diff
@@ -35,6 +35,7 @@ import useUpdatePlayer from './hooks/useUpdatePlayer';
 import MusicPlayView from './MusicPlayView';
 import PackDialog from './PackDialog';
 import Timeline from './Timeline';
+import {useMusicSelector} from './types';
```

Comment:
nit: we can just use the already imported `useAppSelector` - `useMusicSelector` was created before we had the globally typed `useAppSelector`, and we can now replace/deprecate it since we have the global one

Reply:
Ah, good to know.  Updated.